mysql-test/suite/rocksdb/t/add_index_inplace.test

```diff
@@ -342,4 +342,28 @@ while ($i <= $max) {
 
 DROP TABLE t1;
 
+# Test that fast secondary index creation updates cardinality properly
+CREATE TABLE t1 (i INT PRIMARY KEY, j INT) ENGINE = ROCKSDB;
+INSERT INTO t1 VALUES (1,2), (2,4), (3,6), (4,8), (5,10);
+ALTER TABLE t1 ADD INDEX kj(j), ALGORITHM=INPLACE;
```

Comment:
This alter table completes before the flush memtable is called. Can you modify this test so that flush memtable is called while the alter table is running? Basically add a debug_sync_point in the alter path so that it waits on the commit function. Data is in the memtable and the temp key_def is in the hashtable, but the new key definition has not been made permanent in the data dictionary. Trigger a memtable flush at this point. You will need to use multiple connections to do this.